cdap-api/src/main/java/co/cask/cdap/api/schedule/ScheduleSpecification.java

```diff
@@ -27,16 +27,22 @@
  */
 public final class ScheduleSpecification {
   private final Schedule schedule;
+  private final String scheduleType;
```

Comment:
Oops. Didn't know there was `ScheduleSpecificationCodec` already. Will use that.